cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -17,7 +17,9 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-2928/CNV-4166 OVN-Kubernetes network provider
 
-// CNV-4091 nmstate
+* {CNVProductNameStart} uses link:https://nmstate.github.io/[`nmstate`] to report on and configure the state of the node network. This makes it possible to modify network policy configuration, to create Linux bridges, bonds, and VLAN devices on all nodes, by applying a single configuration manifest to the cluster.
```

Comment:
The second sentence is a bit too hard to read. I'd try splitting it into two sentences, and I'd also:
- s/This makes it possible to modify/You can change